lib/typescript/render/providers/chatplugin/ChatPluginRender.tsx

```diff
@@ -1,24 +1,61 @@
 import React from 'react';
-import {isFromContact, Message} from '../../../httpclient/model';
+import {Message} from '../../../httpclient/model';
 import {getDefaultMessageRenderingProps, MessageRenderProps} from '../../shared';
+import {RichCard} from '../../components/RichCard';
+import {RichCardContent, ContentUnion, TextContent} from './chatPluginModel';
 import {Text} from '../../components/Text';
-import {ContentUnion} from './chatPluginModel';
 
 export const ChatPluginRender = (props: MessageRenderProps) => {
-  const message = props.message;
-  return render(mapContent(message), props);
+  const {
+    message: {content},
+    message,
+  } = props;
```

Comment:
nitpich: You can also destructure in the header `({message: {content, message}: MessageRenderProps) => ...`